### ISSUE TITLE
[fast-switcher] Doesn't work in certain situations

### ISSUE THREAD

**Issue by travankor** [WayfireWM/wayfire#1060]
**Describe the bug**
Fast switcher doesn't work in the same places as normal switcher.

**To Reproduce**

1. Enable wm-actions plugins
2. Create view
3. Fullscreen view
4. Create view
5. Create "panel" layer view
6. Try to use fast-switcher

**Expected behavior**
Fast switcher works in the same places as switcher.

**Wayfire version**
0.6.0

**Comment by ammen99** [comment]
Uhh what happens for you? Here it works.

**Comment by travankor** [comment]
Hmm, actually it seems like fast-switcher doesn't work when a layer shell view is there. The animation happens but the focus never changes.

**Comment by ammen99** [comment]
> Hmm, actually it seems like fast-switcher doesn't work when a layer shell view is there. The animation happens but the focus never changes.

Is that layer-shell view possibly with keyboard interactivity enabled?

**Comment by travankor** [comment]
Yes. Also doing the above, causes SSD decoration to indicate multiple views as focused.

**Comment by ammen99** [comment]
> The animation happens but the focus never changes.

Now, this is mandated by layer-shell :)

**Comment by ammen99** [comment]
I think the issues discussed here have been already resolved.
